lib/takeaway.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'order'
+require_relative 'menu'
+require_relative 'message_handler'
+
+class Takeaway
```

Comment:
So this class handles orders? It's always tempting to name classes after some real-world object as you've done with "Takeaway" but it's generally better to go with something more descriptive: "OrderHandler", perhaps.